index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Poetic song lyrics homepage</title>
+</head>
+<body>
+    <header class="intro-header">
+        <ul class="header-ul">
```

Comment:
- [ ] Let's complete the homepage Markup for the mobile as the `more` button is not working yet. Let's make all the songs visible, the partner's section visible, and finally the footer to be visible on the homepage mobile view.

See what you have currently:
<img width="289" alt="Loom_jtF3uYd92x" src="https://user-images.githubusercontent.com/81468675/152468268-261b8c00-dee7-4447-9243-905a494b81ac.png">